openhands/runtime/impl/remote/remote_runtime.py

```diff
@@ -255,6 +256,8 @@ def _parse_runtime_response(self, response: requests.Response):
         start_response = response.json()
         self.runtime_id = start_response['runtime_id']
         self.runtime_url = start_response['url']
+        self.available_hosts = start_response.get('work_hosts', {})
```

Comment:
Did you mean by

```suggestion
        self.available_hosts = start_response.get('web_hosts', {})
```

not sure if it's a typo

Reply:
this is correct. Maybe not the best name but that's what we've got 🙃